src/pkg/services/m365/m365.go

```diff
@@ -135,7 +130,7 @@ func GetUserInfo(
 	ctx context.Context,
 	acct account.Account,
 	userID string,
-) (*UserInfo, error) {
```

Comment:
Out of curiosity, what is the significance of `GetUserInfo` func. And where all it is getting used?

Reply:
SDK users can call it before running a backup for a user to ensure they have up-to-date info about that user.  Supposing they aren't retrieving all users every run, and instead use their own library of user ids that they iterate through to kick off backups, GetUserInfo allows them to check whether the user has services enabled, among other checks, before initializing the backup.